debug-bar-action-and-filters-addon.php

```diff
@@ -182,10 +182,14 @@ function debug_bar_action_and_filters_addon_display_filters() {
 						// Type 2 - closure
 						$table .= '<li>[<em>' . esc_html__( 'closure', 'debug-bar-actions-and-filters-addon' ) . '</em>]</li>';
 						$signature = get_class( $single_function['function'] ) . $hook_in_count;
-					} elseif ( ( is_array( $single_function['function'] ) || is_object( $single_function['function'] ) ) && dbafa_is_closure( $single_function['function'][0] ) ) {
+					} elseif ( is_array( $single_function['function'] ) && is_object( $single_function['function'][0] ) && dbafa_is_closure( $single_function['function'][0] ) ) {
 						// Type 3 - closure within an array
 						$table .= '<li>[<em>' . esc_html__( 'closure', 'debug-bar-actions-and-filters-addon' ) . '</em>]</li>';
 						$signature = get_class( $single_function['function'] ) . $hook_in_count;
+					} elseif ( is_object( $single_function['function'] ) && is_callable( $single_function['function'] ) ) {

```

Comment:
I can't because closure is callable and closures would be caught in Type 9.

I did fix in Type 3 because its detection generates errors on some cases.